packages/wrangler/src/utils/stdout-styling.ts

```diff
@@ -0,0 +1,17 @@
+import chalk from "chalk";
+import supportsColor from "supports-color";
+
+/**
+ * Terminal styling util function that sets text color. If terminal does
+ * not support colors, it will return the unchanged text instead.
+ */
+export function highlight(text: string, colorHex: string): string {
+	return supportsColor.stdout ? chalk.hex(colorHex)(text) : text;
```

Comment:
Doesn't chalk check for this internally anyway?

Reply:
no idea. I'll check